### PR TITLE
experiments/AtmosLES/taylor-green.jl

### PR DESCRIPTION
**Pull request by smarras79**
# Description

New experiment/AtmosLES:

* Taylor-Green vortex decay: isotropic turbulence in a triply periodic domain.

References: 
```
@article{taylorGreen1937,                                                                                                         
author = {Taylor, G. I. and Green, A. E.},                                                                                        
title = {Mechanisms of production of small eddies from large ones},                                                               
journal = {Proc. Roy. Soc. A},                                                                                                    
volume = {158}                                                                                                                    
number = {895},                                                                                                                   
year = {1937},                                                                                                                    
doi={doi.org/10.1098/rspa.1937.0036},                                                                                             
}                    

@article{rafeiEtAl2018,                                                                                                           
author = {Rafei, M.E. and K\"on\"oszy, L. and Rana, Z.},                                                                          
title = {Investigation of Numerical Dissipation in Classicaland Implicit Large Eddy Simulations,                                  
journal = {Aerospace},                                                                                                            
year = {2018},                                                                                                                    
}                                                                                                                                 
                                                                                                                                  
@article{bullJameson2014,                                                                                                         
author = {Bull, J.R. and Jameson, A.},                                                                                            
title = {Simulation of the Compressible {Taylor Green} Vortex                                                                     
using High-Order Flux Reconstruction Schemes},                                                                                    
journal = {AIAA Aviation 7th AIAA theoretical fluid mechanics conference},                                                        
year = {2014},                                                                                                                    
}  
```       

<!--- Please fill out the following section --->

I have

- [ ] Written and run all necessary tests with CLIMA by including `tests/runtests.jl`
- [ ] Followed all necessary [style guidelines](https://CliMA.github.io/CLIMA/latest/CodingConventions.html) and run `julia .dev/format.jl`
- [ ] Updated the documentation to reflect changes from this PR.

<!--- Please leave the following section --->

# For review by CLIMA developers

- [ ] There are no open pull requests for this already
- [ ] CLIMA developers with relevant expertise have been assigned to review this submission
- [ ] The code conforms to the [style guidelines](https://CliMA.github.io/CLIMA/latest/CodingConventions.html) and has consistent naming conventions. `julia .dev/format.jl` has been run in a separate commit.
- [ ] This code does what it is technically intended to do (all numerics make sense physically and/or computationally)

### CODE REVIEW

File: experiments/AtmosLES/taylor-green.jl [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+@article{bullJameson2014,
+author = {Bull, J.R. and Jameson, A.},
+title = {Simulation of the Compressible {Taylor Green} Vortex
+using High-Order Flux Reconstruction Schemes},
+journal = {AIAA Aviation 7th AIAA theoretical fluid mechanics conference},
+year = {2014},
+}
+
+"""
+
```
Comment (charleskawczynski) [inline experiments/AtmosLES/taylor-green.jl:70]:
```suggestion
```

Comment (charleskawczynski) [reply experiments/AtmosLES/taylor-green.jl:70]: (Doc strings cannot have newlines between methods/structs)

**Comment by smarras79** [comment]
bors try

**Pull request merged**